templates/vue/src/components/NodeModal.vue

```diff
@@ -418,6 +431,17 @@ export default {
         }
       },
     },
+    hasSubmissionApiError() {
+      if (this.apiError) {
+        this.submissionError =
+          "An unexpected error occurred and the node could not be submitted"
```

Comment:
Why don't we show the actual API error instead rather than this ambiguous message?